<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>task_edit</class>
 <widget class="QMainWindow" name="task_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="time_show">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 10pt &quot;Segoe UI Variable Small Semibol&quot;;
border-image: url(:/new/prefix1/labels/11.jpg);</string>
    </property>
    <property name="text">
     <string>timeshow</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>33</y>
      <width>461</width>
      <height>582</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;华文琥珀&quot;;
border-image: url(:/new/prefix1/labels/10.jpg);</string>
       </property>
       <property name="text">
        <string>你的计划</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;黑体&quot;;
border-image: url(:/new/prefix1/labels/4.jpg);</string>
       </property>
       <property name="text">
        <string>请输入你的计划。</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="input_name">
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Times New Roman&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;黑体&quot;;
border-image: url(:/new/prefix1/labels/4.jpg);</string>
       </property>
       <property name="text">
        <string>请描述计划的详细内容。</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="input_content">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;华文琥珀&quot;;
border-image: url(:/new/prefix1/labels/10.jpg);</string>
       </property>
       <property name="text">
        <string>你的记录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;黑体&quot;;
border-image: url(:/new/prefix1/labels/4.jpg);</string>
       </property>
       <property name="text">
        <string>请输入你的记录。</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="input_name2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;黑体&quot;;
border-image: url(:/new/prefix1/labels/4.jpg);</string>
       </property>
       <property name="text">
        <string>请描述记录的详细内容。</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="inputcontent_2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>160</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="complete">
       <property name="font">
        <font>
         <family>华文新魏</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/labels/4.jpg);</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sweep">
       <property name="font">
        <font>
         <family>华文新魏</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/prefix1/labels/4.jpg);</string>
       </property>
       <property name="text">
        <string>Delete_all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>华文琥珀</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;华文琥珀&quot;;
border-image: url(:/new/prefix1/labels/5.jpg);</string>
       </property>
       <property name="text">
        <string>是否需要提醒？</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="remindput">
       <property name="font">
        <font>
         <family>华文新魏</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
